fhir-bulkdata-webapp/src/main/java/com/ibm/fhir/bulkdata/export/writer/SparkParquetWriter.java

```diff
@@ -35,6 +35,7 @@ public SparkParquetWriter() {
             .appName("parquetWriter")
             .master("local[*]")
             .config("spark.ui.enabled", false)
+            .config("spark.driver.port", "38000")
```

Comment:
@lmsurpre  FYI - we had a port conflict with the berkley db and had to assign a unique port